functions/src/events/Event.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Controller, Post, Query, Route } from 'tsoa';
+import { add } from 'typesaurus';
+import { EventsCollection } from '../db/collections';
+
+@Route('events')
+export class EventsController extends Controller {
+  /**
+   * Retrieves all the courses available. If query params are passed in, they will be used to filter results.
+   * @param subject
+   * @param code
+   */
+  @Post()
+  public async postEvent(@Query() id: string): Promise<{ success: boolean }> {
```

Comment:
is `id` the best name? Not sure what you had in mind for intended use but if we're passing something like `Contest page visit` or something perhaps just `event` is better

Reply:
`id` just maps to what the URLs query params are where `id` is actually the location. I'll change it .